spec/mainadditions.html

```diff
@@ -271,6 +271,266 @@ <h1>AvailableNamedTimeZoneIdentifiers ( ): a List of Time Zone Identifier Record
+        </p>
+      </emu-note>
+      <emu-note>
+        <p><emu-eqn>UTC(LocalTime(_t_<sub>UTC</sub>))</emu-eqn> is not necessarily always equal to <emu-eqn>_t_<sub>UTC</sub></emu-eqn>. Correspondingly, <emu-eqn>LocalTime(UTC(_t_<sub>local</sub>))</emu-eqn> is not necessarily always equal to <emu-eqn>_t_<sub>local</sub></emu-eqn>.</p>
+      </emu-note>
+    </emu-clause>
+
+    <p>[...]</p>
+
+    <emu-clause id="sec-timezoneestring" type="abstract operation">
+      <h1>
+        TimeZoneString (
+          _tv_: an integral Number,
+        ): a String
+      </h1>
+      <dl class="header">
+      </dl>
+      <emu-alg>
+        1. Let _systemTimeZoneIdentifier_ be SystemTimeZoneIdentifier().
+        1. <ins>Let _parseResult_ be ParseTimeZoneIdentifier(_systemTimeZoneIdentifier_).</ins>
+        1. If <del>IsTimeZoneOffsetString(_systemTimeZoneIdentifier_) is *true*</del><ins>_parseResult_.[[OffsetMinutes]] is not ~empty~</ins>, then
+          1. Let _offsetNs_ be <del>ParseTimeZoneOffsetString(_systemTimeZoneIdentifier_)</del><ins>_parseResult_.[[OffsetMinutes]] × (6 × 10<sup>10</sup>)</ins>.
+        1. Else,
+          1. Let _offsetNs_ be GetNamedTimeZoneOffsetNanoseconds(_systemTimeZoneIdentifier_, ℤ(ℝ(_tv_) × 10<sup>6</sup>)).
+        1. Let _offset_ be 𝔽(truncate(_offsetNs_ / 10<sup>6</sup>)).
+        1. If _offset_ is *+0*<sub>𝔽</sub> or _offset_ > *+0*<sub>𝔽</sub>, then
+          1. Let _offsetSign_ be *"+"*.
+          1. Let _absOffset_ be _offset_.
+        1. Else,
+          1. Let _offsetSign_ be *"-"*.
+          1. Let _absOffset_ be -_offset_.
+        1. Let _offsetMin_ be ToZeroPaddedDecimalString(ℝ(MinFromTime(_absOffset_)), 2).
+        1. Let _offsetHour_ be ToZeroPaddedDecimalString(ℝ(HourFromTime(_absOffset_)), 2).
+        1. Let _tzName_ be an implementation-defined string that is either the empty String or the string-concatenation of the code unit 0x0020 (SPACE), the code unit 0x0028 (LEFT PARENTHESIS), an implementation-defined timezone name, and the code unit 0x0029 (RIGHT PARENTHESIS).
+        1. Return the string-concatenation of _offsetSign_, _offsetHour_, _offsetMin_, and _tzName_.
+      </emu-alg>
+    </emu-clause>
+
+    <emu-clause id="sec-isoffsettimezoneidentifier" oldids="sec-istimezoneoffsetstring" type="abstract operation">
+      <h1>
+        IsOffsetTimeZoneIdentifier (
+          _offsetString_: a String,
+        ): a Boolean
+      </h1>
+      <dl class="header">
+        <dt>description</dt>
+        <dd>The return value indicates whether _offsetString_ conforms to the grammar given by <del>|UTCOffset|</del><ins>|TimeZoneUTCOffsetName|</ins>.</dd>
+      </dl>
+      <emu-note type="editor">
+        This abstract operation was renamed from IsTimeZoneOffsetString.
+        After https://github.com/tc39/ecmarkup/issues/530 is fixed, this note can be replaced with `ins` and `del` in the AO name above.
+      </emu-note>
```

Comment:
Fixed now in ecmarkup v17.0.1!
https://github.com/tc39/ecmarkup/pull/533#issuecomment-1596272123

Reply:
I don't think this is actually fixed, at least I couldn't figure out how to make it work: https://github.com/tc39/ecmarkup/pull/533#issuecomment-1596453212